src/domain/reports/glass-admin/usecases/UpdateGLASSDataMaintenanceUseCase.ts

```diff
@@ -0,0 +1,17 @@
+import { UseCase } from "../../../../compositionRoot";
+import { GLASSDataMaintenanceRepository } from "../repositories/GLASSDataMaintenanceRepository";
+
+export class UpdateGLASSDataMaintenanceUseCase implements UseCase {
```

Comment:
We used to do this "implements UseCase", but we have noticied it hinders code navigation while adding very little value, so better just to remove it.